tests-config.json

```diff
@@ -0,0 +1,2703 @@
+
+type Test = {
+    testFile?: string;
+    isWs?: boolean;
+    public?: boolean;
+    args?: any[];
+    skippedProperties?: {}
+    skip?: string;
+}
+
+type Tests = Test & {
+    skip?: string;
+    skipWs?: string;
+    skipPhpAsync?: string;
+    until?: string;
+    httpProxy?: string;
+    httpsProxy?: string;
+    wsProxy?: string;
+    wssProxy?: string;
+    timeout?: number;
+}
+
+type TestConfig = {
+    [key: string]: Tests;
+}
+
+// *********************************
+// ***** AUTO-TRANSPILER-START *****
+function config (symbol: string = 'BTC/USDT', code: string = 'USDT', isSpot: boolean = true) {
+    return ({
+        'exchange': {
+            // public tests
+            'loadMarkets': {
+                'testFile': 'loadMarkets',
```

Comment:
no need to repeat the method names as testfiles, it convults the json (at this stage).  in most cases, they will have same string value, so let's leave `testFile` key just for override if the key name does not match test file name.

Reply:
This was done on purpose. The key is the test name that can be any string to describe the test, the testFile is the name of file to run